okhttp-tests/src/test/java/com/squareup/okhttp/internal/http/ThreadInterruptTest.java

```diff
@@ -70,6 +92,35 @@ protected void configureSocket(Socket socket) throws IOException {
     });
   }
 
+  @Test @Ignore public void interruptConnect() throws Exception {
```

Comment:
Is this true? What about when building an tunnel for HTTPS through an HTTP proxy? That'll fire during InputStream.read().